app/controllers/taxons_controller.rb

```diff
@@ -16,6 +16,17 @@ def create
     redirect_to taxons_path
   end
 
+  def show
+    @taxon = TaxonForm.build(content_id: params[:id])
+    # FIXME: ideally the gds-api-adapters should be updated to include support
```

Comment:
I've added support to gds-api-adapters in a branch. Can you test it with this branch to make sure that works?

https://github.com/alphagov/gds-api-adapters/tree/fix-support-for-types